argparse.h

```diff
@@ -124,6 +126,7 @@ void argparse_describe(struct argparse *self, const char *description,
 int argparse_parse(struct argparse *self, int argc, const char **argv);
 void argparse_usage(struct argparse *self);
 
+
```

Comment:
Was this intended? Looks unrelated to the commit to me. So if this line is missing to satisfy some coding style or so, I'd suggest putting it in a separate commit to allow reverting the introduction of `argparse_help_cb_no_exit` (if needed) without undoing this change.

Reply:
This was unintended.

Reply:
Then I suggest dropping it from this commit. :wink: